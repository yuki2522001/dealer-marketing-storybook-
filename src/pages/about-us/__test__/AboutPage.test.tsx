import { render } from "@testing-library/react";
import AboutPage from "../index";

describe("AboutPage", () => {
  test("Should render AboutPage", () => {
    const { getByTestId } = render(<AboutPage />);
    const aboutPage = getByTestId("layout");
    expect(aboutPage).toBeInTheDocument();
  });

  test("Matches snapshot", () => {
    const { container } = render(<AboutPage />);
    expect(container).toMatchSnapshot();
  });
});
