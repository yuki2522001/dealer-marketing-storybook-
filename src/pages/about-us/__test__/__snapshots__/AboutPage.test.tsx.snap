// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Matches snapshot 1`] = `
<div>
  <main
    data-testid="layout"
  >
    <header
      data-testid="header"
    >
      <div>
        <ul
          data-testid="menu"
        >
          <li>
            <a
              href="/expert-panel"
            >
              Our Experts
            </a>
          </li>
          <li>
            <a
              href="/about-us"
            >
              About
            </a>
          </li>
        </ul>
      </div>
    </header>
    <nav
      data-testid="navigation"
    >
      <div
        data-testid="logo"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27256%27%20height=%2743%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="This is logo"
            data-nimg="intrinsic"
            decoding="async"
            href="/"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; background-size: cover; background-position: 0% 0%; filter: blur(20px); background-image: url(/images/backgrounds/blur.jpg);"
          />
          <noscript />
        </span>
      </div>
      <div>
        <ul
          data-testid="menu"
        >
          <li>
            <a
              href="/"
            >
              Research
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Best practice
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Commentary
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Reviews
            </a>
          </li>
        </ul>
        <i
          data-testid="icon"
        />
      </div>
    </nav>
    <div
      data-testid="banner"
    >
      <div>
        <span
          style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
        >
          <img
            alt="This is image of banner"
            data-nimg="fill"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; background-size: cover; background-position: 0% 0%; filter: blur(20px); background-image: url(/images/backgrounds/blur.jpg);"
          />
          <noscript />
        </span>
      </div>
      <h1>
        About Us
      </h1>
    </div>
    <div
      class="container"
      data-testid="about-section"
    >
      <div>
        <h2>
          The DMM Expert Panel
        </h2>
        <p
          data-testid="text"
        >
          Dealer Marketing Magazine is a leading online trade publication that speaks to a large online audience of dealers and auto industry professionals. Our focus is on editorial content that educates and enlightens auto dealers using the latest research, how-to, and thought leadership articles.
        </p>
        <p
          data-testid="text"
        >
          The magazine’s journey began in 2002 by offering syndicated automotive television commercials that dealers could use in their own markets. After successfully running a printed magazine for 16 years, the magazine is now a digital platform offering dealers expert automotive knowledge at their fingertips.
        </p>
        <p
          data-testid="text"
        >
          We believe that dealers are going through an enormous transformation in this current whirlwind digital era, and we are here to support them by providing new tech, trends, research, and strategies used by automotive industry leaders to catapult business growth and success.
        </p>
        <h2>
          WHAT WE TALK ABOUT
        </h2>
        <p
          data-testid="text"
        >
          Our experience suggests that dealers are eager to understand the evolving 21st century customer journey as well as the competitive nature thereof and are interested in learning how to be more effective marketers.
        </p>
        <p
          data-testid="text"
        >
          In order to stay competitive, dealers need to gear up and embrace the future of automotive. A provider of fresh viewpoints, Dealer Marketing Magazine delivers exceptional insight focused on The Customer Journey and The Dealership Business.
        </p>
        <h2>
          THE EXPERTS
        </h2>
        <p
          data-testid="text"
        >
          Our Expert Panel consists of thirty passionate automotive industry experts who share their knowledge, research, and experience to help keep dealers informed on various topics such as Marketing & Advertising, Lead Generation, F&I, Digital Retailing, Leadership & Management, Mobility, and more.
        </p>
        <h2>
          STAY CONNECTED
        </h2>
        <p
          data-testid="text"
        >
          Want to stay up to date with the latest automotive Best Practices, Commentary, Research, and Events? Sign up for our newsletter here to receive some new perspectives and grow your network.
        </p>
        <h2>
          THE EXPERTS
        </h2>
        <p
          data-testid="text"
        >
          To list your automotive event for free, contact us at events@dealermarketing.com
        </p>
      </div>
    </div>
    <footer
      data-testid="footer"
    >
      <ul
        data-testid="menu"
      >
        <li>
          <a
            href="/expert-panel"
          >
            Our Experts
          </a>
        </li>
        <li>
          <a
            href="/about-us"
          >
            About
          </a>
        </li>
      </ul>
      <div>
        <ul
          data-testid="menu"
        >
          <li>
            <a
              href="/"
            >
              Privacy policy
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Terms of service
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              Advertise
            </a>
          </li>
        </ul>
        <div>
          <p
            data-testid="text"
          >
            Copyright © 2022 Dealer Marketing Magazine
          </p>
        </div>
      </div>
    </footer>
  </main>
</div>
`;
